From 49a14e693124dc34f2cccbfb755d01a1198aa6bf Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarsk@gmail.com>
Date: Wed, 19 Jul 2023 15:24:22 -0700
Subject: [PATCH] disable pre-canned sysroot

Override should be enabled if a custom sysroot using --target-sysroot is
specified. If --target-sysroot is not set, then it should default to the
pre-canned sysroot.

Upstream: https://github.com/flutter/flutter/issues/123073
Signed-off-by: Joel Winarske <joel.winarsk@gmail.com>
Signed-off-by: Adam Duskett <adam.duskett@amarulasolutions.com>
---
 build/config/sysroot.gni | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/config/sysroot.gni b/build/config/sysroot.gni
index 7987e519d..1de694263 100644
--- a/build/config/sysroot.gni
+++ b/build/config/sysroot.gni
@@ -14,7 +14,7 @@ declare_args() {
 
   # Whether to use the default sysroot when building for Linux, if an explicit
   # sysroot isn't set.
-  use_default_linux_sysroot = true
+  use_default_linux_sysroot = false
 }
 
 if (current_toolchain == default_toolchain && target_sysroot != "") {
-- 
2.41.0

